### PR TITLE
fix: Docs to Yaml

### LINKED ISSUE #370: Convert Docs to Yaml

### PR DESCRIPTION
**Pull request by zagronitay**
#### Summary

closes #370 

---

Use the following steps to ensure your PR is ready to be reviewed

- [ ] Read the [contribution guidelines](../blob/main/CONTRIBUTING.md) 🧑‍🎓
- [ ] Run `go fmt` to format your code 🖊
- [ ] Lint your changes via `golangci-lint run` 🚨 (install golangci-lint [here](https://golangci-lint.run/usage/install/#local-installation))
- [ ] Update or add tests. Learn more about testing [here](https://docs.cloudquery.io/docs/developers/sdk/testing) 🧪
- [ ] Update the docs by running `go run ./docs/docs.go` and committing the changes 📃
- [ ] If adding a new resource, add [relevant Terraform files](../terraform) in a separate PR 📂
- [ ] Ensure the status checks below are successful ✅

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -40,5 +40,6 @@ __debug_bin
 .env
 terraform/downloaded*
 config.hcl
+config.yml
```
Comment (bbernays) [inline .gitignore:43]: I think this should be `cloudquery.yml`

File: test/policy_cq_config.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+cloudquery:
```
[the remark this answers is not in the document]
Comment (shimonp21) [reply test/policy_cq_config.yml:1]: Probably, but moving it is out of scope for this PR, no?

[the remark this answers is not in the document]
Comment (yevgenypats) [reply test/policy_cq_config.yml:1]: I think it should be under test as there are tests that use it and they are unrelated to github workflows.

**Review by yevgenypats — APPROVED** [review]

File: test/policy_cq_config.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    type: postgres
+    username: postgres
+    password: pass
+    host: localhost
+    port: 5432
+    database: postgres
+    sslmode: disable
+providers:
+  # provider configurations
+  azure:
```
Comment (erezrokah) [inline test/policy_cq_config.yml:15]: The format should be:
```yaml
providers:
  - name: azure
  ...
```

**Pull request merged**